jwst/assign_wcs/assign_wcs.py

```diff
@@ -62,6 +63,9 @@ def load_wcs(input_model, reference_files={}, nrs_slit_y_range=None):
         wcs = WCS(pipeline)
         output_model.meta.wcs = wcs
         output_model.meta.cal_step.assign_wcs = 'COMPLETE'
+        output_model.meta.source_catalog.filename = os.path.basename(
+            output_model.meta.source_catalog.filename)
```

Comment:
This meta attribute is only going to be populated for exposures containing WFSS (grism) data; all others will be None. So a check is needed here to see if it exists. That's what's causing all the Travis errors.

Reply:
Actually, this meta attribute is first populated in the `source_catalog/source_catalog_step.py` module. That's where it should be fixed to not include the full path when it gets populated. Of course it doesn't hurt to have another check here, in case, for example, a user populates it by hand with a full path included.